hoomd/md/IntegratorTwoStep.cc

```diff
@@ -112,6 +108,7 @@ void IntegratorTwoStep::update(uint64_t timestep)
         method->includeRATTLEForce(timestep + 1);
         }
 
+    updateRigidBodies(timestep + 1);
```

Comment:
This is right to have here, but we need to be careful because of the comment below. In MPI, we will need to force a ghost particle update.